spec/buffered-process-spec.coffee

```diff
@@ -117,6 +111,7 @@ describe "BufferedProcess", ->
       expect(stdout).toEqual ''
 
   it "calls the specified stdout callback only with whole lines", ->
+    # WINSPEC: Fails because command is too long, /bin/echo is *nix only, odd newlining and quoting
```

Comment:
I tried to make it run echo on Windows as well by specifying 'shell' but the line length was too short. I reduced that and then quoting was off and it only returned the first block.  

I've been trying to think what would return a lot of lines consistently between the two - apm perhaps? Not sure that's always enabled at this point.  It's difficult to come up with a command on Windows that will return multiple lines consistently regardless of Windows version or language.  Open to suggestions.